compress/lib.h

```diff
@@ -30,6 +30,18 @@
  * | compress/lz4.c      | @subpage compress_lz4  |
  * | compress/zlib.c     | @subpage compress_zlib |
  * | compress/zstd.c     | @subpage compress_zstd |
+ *
+ * Default usage with Compression Level set to 1:
+ * open() -> N times compress() -> close()
+ * open() -> N times decompress() -> close()
+ *
+ * Usage with Compression Level set to N:
+ * open() -> set_level(N) -> N times compress() -> close()
+ * open() -> N times decompress() -> close()
+ *
+ * Usage with Zstd Dictionary Compression and Level:
+ * open() -> set_level(N) -> set_dictionary() -> N times compress() -> close()
+ * open() -> set_dictionary() -> N times decompress() -> close()
```

Comment:
```suggestion
 * Default usage with Compression Level set to 1:
 * - open() -> N times compress() -> close()
 * - open() -> N times decompress() -> close()
 *
 * Usage with Compression Level set to N:
 * - open() -> set_level(N) -> N times compress() -> close()
 * - open() -> N times decompress() -> close()
 *
 * Usage with Zstd Dictionary Compression and Level:
 * - open() -> set_level(N) -> set_dictionary() -> N times compress() -> close()
 * - open() -> set_dictionary() -> N times decompress() -> close()
```

This comment is in markdown, so I've added some bullet points to prevent it being wrapped into three long sentences.